docs/grammar.txt

```diff
@@ -72,8 +75,11 @@ VariableDefinition = ('var' IdentifierList | VariableDeclaration) ( '=' Expressi
 IdentifierList = '(' ( Identifier? ',' )* Identifier? ')'
 
 // Precedence by order (see github.com/ethereum/solidity/pull/732)
-Expression =
-  ( Expression ('++' | '--') | FunctionCall | IndexAccess | MemberAccess | '(' Expression ')' )
+Expression
+  = Expression ('++' | '--')
```

Comment:
Turns out this is not even documented in `miscellaneous.rst:precedence`. Would you be able to confirm the precedence order, update in the doc and the grammar accordingly?